File: scala/2017/2/solution1.scala
import scala.io.Source

val lines = Source.fromFile("../../../data/2017/2/data.txt").getLines.toList

var total = 0
for (line <- lines) {
  total += line.split("\\s").map(_.toInt).max - line.split("\\s").map(_.toInt).min
}

println(f"Solution is $total")

File: julia/2017/2/solution1.jl
f = open("../../../data/2017/2/data.txt")
lines = readlines(f)
close(f)

total = 0
for line in lines
    splitline = map(x->parse(Int,x), split(line, "\t"))
    total += maximum(splitline) - minimum(splitline)
end

println("Solution is $total")
